src/condition.h

```diff
@@ -429,4 +429,27 @@ class ConditionSpellGroupCooldown final : public ConditionGeneric
 		}
 };
 
+class ConditionDrunk final : public Condition
+{
+	public:
+		ConditionDrunk(ConditionId_t id, ConditionType_t type, int32_t ticks, bool buff, uint32_t subId, uint32_t drunkenness, bool aggressive = false) :
+			Condition(id, type, ticks, buff, subId, aggressive),
+			drunkenness(drunkenness == 0 ? 25 : drunkenness) {}
```

Comment:
I still think it would be better to move the assignment to scope and use something like
```
if (drunkenness != 0) {
     this->drunkenness = drunkeness;
}
```

Reply:
Hard agree. Its more clear that way.

I just updated the branch with the changes.